cloudpub/ms_azure/utils.py

```diff
@@ -281,6 +281,12 @@ def get_skuid(arch):
             return plan_name
         return f"{plan_name}-{arch.lower()}"
 
+    def get_safe_security_type(image_type):
+        # Arches which aren't x86Gen2 (like ARM64) doesn't work well with security type
+        if image_type != "x64Gen2":
+            return None
+        return security_type
+
```

Comment:
The parameter ` security_type: Optional[List[str]] = None` will be noop after this change.

Can we remove it from the `_build_skus`?

Reply:
It won't, because the `security_type` returned in the line 288 is the one from the parameter... The `get_safe_security_type` is a nested function... Do you prefer I do it other way? Like with a private function?